src/data/real/sqrt.lean

```diff
@@ -45,6 +45,12 @@ order_iso.symm $ strict_mono.order_iso_of_surjective (λ x, x * x)
   (continuous_id.mul continuous_id).surjective tendsto_mul_self_at_top $
     by simp [order_bot.at_bot_eq]
 
+lemma sqrt_le_sqrt_iff : sqrt x ≤ sqrt y ↔ x ≤ y :=
```

Comment:
I think it's quite common to drop the `_iff` part. (Also elsewhere.)
```suggestion
lemma sqrt_le_sqrt : sqrt x ≤ sqrt y ↔ x ≤ y :=
```

Reply:
Yep, but `real.sqrt_le_sqrt` is already the (stronger) one way implication and I thought I would at least keep `nnreal.sqrt_le_sqrt_iff` and `real.sqrt_le_sqrt_iff` consistent.